alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -185,6 +185,21 @@ class MonitorRunner(
     }
 
     suspend fun runMonitor(monitor: Monitor, periodStart: Instant, periodEnd: Instant, dryrun: Boolean = false): MonitorRunResult {
+        /*
```

Comment:
Looks like you picked inject roles changes. Do we need this change for getDest, getAlerts?

Reply:
Removed. I accidentally picked it up